e2e-test/20220705.demo1/README.md

```diff
@@ -0,0 +1,148 @@
+# Demo 1 - July, 5th 2022
+
+This test demonstrates user story **As a data intermediary I want to request the fragmented GIPOD LDES data set.** (VSDSPUB-62).
+
+> **Note**: currently, there is no automated test for this demo.
+
+This LDES client E2E test is similar to the [second demo given on May, 24th](../20220524.demo-2/README.md).
+The same context applies.
+
+To demonstrate the correct working of the LDES client you need to:
+
+1. [Start systems](#start-systems).
+2. [Setup demo](#setup-demo).
+3. [Scenario: replication has not yet started](#TODO)
+4. [Scenario: replication has just started](#TODO)
+5. [Scenario: data set contains 2 fragments](#TODO)
+6. [Scenario: data set contains 3 fragments](#TODO)
+7. [Stop systems](#stop-systems)
```

Comment:
In every demo scenario parts of the documentation are identical.  Maybe it can be good to have a centralized e2e documentation folders that contains different md files that describe common used documentation. Then one can just reference them in the specific README.md for a demo. This will reduce work and reduce fixes in describing certain parts.